src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/ReportService.kt

```diff
@@ -216,6 +217,36 @@ class ReportService(
     }.getOrNull()
   }
 
+  @Transactional
+  fun addDescriptionAddendum(id: UUID, descriptionAddendum: DescriptionAddendum): ReportWithDetails? {
+    return reportRepository.findOneEagerlyById(id)?.let { reportEntity ->
+      reportEntity.addDescriptionAddendum(
+        createdBy = descriptionAddendum.createdBy,
```

Comment:
you are going to pass the username who created then?  Rather than use the authenticationHolder.username?

Reply:
i thought about doing this in future pr #329 but haven't yet… maybe i will have a go…